great_expectations/data_context/templates.py

```diff
@@ -4,15 +4,11 @@
 PROJECT_HELP_COMMENT = """
 # Welcome to Great Expectations! Always know what to expect from your data.
 # 
-# This project configuration file allows you to define datasources, generators,
-# integrations, and other things to help you crush pipeline debt.
+# Here you can define datasources, generators, integrations and more. This file
+# is intended to be committed to your repo. For help with configuration please:
+#   - Read our docs: https://docs.greatexpectations.io/en/""" + rtd_url_ge_version + """/core_concepts/data_context.html#configuration
+#   - Join our slack channel: http://greatexpectations.io/slack
 #
-# This file is intended to be committed to your source control repo.
-
-# For more help configuring great expectations see the documentation at:
-# https://docs.greatexpectations.io/en/""" + rtd_url_ge_version + """/core_concepts/data_context.html#configuration
```

Comment:
This docs URL does not exist anymore - should be changed to "reference/data_context_reference.html#configuration"

Reply:
Fixed.